.changeset/pretty-countries-punch.md

```diff
@@ -0,0 +1,6 @@
+---
+'@microsoft/atlas-site': minor
+'@microsoft/atlas-css': minor
+---
+
+Adding color theme to gradient component and updating documentation
```

Comment:
Nit: Could this be updated to something like:

```suggestion
Adding radiant option to gradient component and updating documentation
```

Reply:
Makes sense, committed suggestion.